qdrant-landing/content/articles/scalar-quantization.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Quantized Qdrant Queries Quality
+short_description: "TBD"
+description: "TBD"
+social_preview_image: /articles_data/scalar-quantization/social_preview.png
+small_preview_image: /articles_data/scalar-quantization/quantization-icon.svg
+preview_dir: /articles_data/scalar-quantization/preview
+weight: 3
+author: XXX
+author_link: XXX
+date: 2023-03-21T10:45:00+01:00
+draft: false
+keywords:
+  - vector search
+  - scalar quantization
+  - TBD
+---
+
+High-dimensional vector embeddings can be memory-intensive, especially when working with 
+large datasets consisting of millions of vectors. Memory footprint really starts being 
+a concern when we scale things up. A small choice of the data type used to store a single
```

Comment:
maybe `small` -> `simple`?

As I understood from https://context.reverso.net/translation/english-polish/A+small+choice

It rather means a small variety of options